markdown/chapters/appendices/playables.md

```diff
@@ -0,0 +1,278 @@
+are definitely ways that you can capitalise on the surprise factor as a result
+of it but do be aware, one of the pitfalls in running such a sideboard plan can
+be an over-reliance on transforming and a lack of trust/extreme dilution of the
+plan A.
+
+There are ways you can focus more on the creature plan with the early 2022
+dubbing of *Tempo Doomsday* lists but be warned it will skew certain matchups
+out of favour and sometimes result in very clunky hands or gameplans with the
+wrong parts of both sides being found.
+
+## Multi-Draw Spells
+
+:::row{variant=centered}
+- Predict
+- Night's Whisper
+- Expressive Iteration
+- Reckless Impulse
+:::
+
+Traditionally :card[Ideas Unbound] is the *multi-draw* tool that is used to try
+and dig multiple cards into a pile whilst enabling added protection in your
+hands or the greatest resource efficiency. Outside of the combo however, the
+card is somewhat lackluster and thus many look towards arguably *better* cards
+in a vacuum to take the slot.
+
+:card[Predict] is the most popular of these and has been shown to be perfectly
+viable as an alternative with plenty of historical use examples from 2020
+onwards. The main positives include synergising really well with the cantrip
+shell of the deck and being able to facilitate a longer game. The issues with it
+include the fact it only draws 2 cards, not 3, and it both risks a card to the
+graveyard and relies on the top of your library in the face of many shuffle
+effects like :card[Surgical Extraction] or :card[Boseiju, Who Endures] which can
+result in hitting Oracle itself or forcing a turn pass you can't afford.
+It's very viable but is likely slightly weaker in a dedicated Doomsday list.
+
+:card[Night's Whisper] is another commonly suggested alternative. It pairs very
+strongly with excess {B} mana from ritual effects and is a strong card in its
+own right pre-combo. Unfortunately only drawing two cards plus having life loss
+after Doomsday resolves can be limiting and costly, especially if having to be
+paired with Street Wraith cycles or Force pitch casting. It's a fine card but
+normally gets paired alongside Ideas Unbound rather than replacing it.
+
+:card[Expressive Iteration] is an incredibly potent card with it allowing
+selective filtering and card advantage. Because of the way it is worded you
+could in theory use it to dig into a pile and draw LED, exiling something to
+then be able to cast with said LED. Generally though the mana cost and specific
+wording of it is a little restrictive. Making {U}{R} in a deck that wants to go
+from {B}{B}{B} to {U}{U} can be tricky, especially with so many mana denial
+elements omni-present in the format. The order of operations on the card also
+mean that if you are in a position to cast it with only two cards left in your
+deck, you would put one into your hand and one back into the deck but exile
+none. An excellent card for pre-combo play but not brilliant for after Doomsday
+resolves.
+
```

Comment:
overall the multi draw spells are really bad in times where we want to go fast. You only need the one in your pile, probably IU